src/MoBi.Presentation/Presenter/ObjectPathCreatorAtTransport.cs

```diff
@@ -35,15 +35,21 @@ public ObjectPathCreatorAtTransport(IObjectPathFactory objectPathFactory, IAlias
       /// </returns>
       public override ReferenceDTO CreatePathFromParameterDummy(ObjectBaseDTO objectBaseDTO, bool shouldCreateAbsolutePaths, IEntity refObject, IUsingFormula editedObject)
       {
-         var dtoDummyParameter = (DummyParameterDTO) objectBaseDTO;
+         var dtoDummyParameter = (DummyParameterDTO)objectBaseDTO;
          var firstPathElement = getPathKeywordForContainer(dtoDummyParameter.Parent.ParentContainer);
-         if (firstPathElement.IsNullOrEmpty())
-            return base.CreatePathFromParameterDummy(objectBaseDTO, shouldCreateAbsolutePaths, refObject, editedObject);
+         if (firstPathElement.IsNullOrEmpty() || shouldCreateAbsolutePaths)
+         {
+            var dto = base.CreatePathFromParameterDummy(objectBaseDTO, shouldCreateAbsolutePaths, refObject, editedObject);
+            if (!shouldCreateAbsolutePaths)
+               dto.Path.Replace(dtoDummyParameter.ModelParentName, ObjectPathKeywords.MOLECULE);
+
+            return dto;
+         }
 
          var moleculeElement = shouldCreateAbsolutePaths ? dtoDummyParameter.ModelParentName : ObjectPathKeywords.MOLECULE;
```

Comment:
I'm a bit confused about this because the line here has not been changed by this PR. However, the `shouldCreateAbsolutePaths` will always be false by this point. Is it the case that the conditional here is not required at all now?

If so, you can remove it.